Core/include/Acts/Geometry/TrackingVolume.hpp

```diff
@@ -503,6 +503,10 @@ class TrackingVolume : public Volume {
   /// @param policy is the navigation policy to be registered
   void setNavigationPolicy(std::unique_ptr<INavigationPolicy> policy);
 
+  /// Return the navigation policy
+  /// @return the navigation policy
+  const INavigationPolicy* navigationPolicy() const;
```

Comment:
I'm also adding this in https://github.com/acts-project/acts/pull/4444 for thew detray conversion